spec/formatting.md

```diff
@@ -237,23 +236,23 @@ Otherwise, the _variable_ is an implicit reference to an input value,
 and its value is looked up from the _formatting context_ _input mapping_.
 
 The resolution of a _variable_ fails if no value is identified for its _name_.
-If this happens, an _Unresolved Variable_ error is emitted.
-If a _variable_ would resolve to a _fallback value_,
-this MUST also be considered a failure.
+If this happens, an _Unresolved Variable_ error is emitted
+and a _fallback value_ is used as _resolved value_ of the _variable_.
 
 ### Function Resolution
 
 To resolve an _expression_ with a _function_,
 the following steps are taken:
 
 1. If the _expression_ includes an _operand_, resolve its value.
-   If this fails, use a _fallback value_ for the _expression_.
+   If this is a _fallback value_,
+   use it as the _resolved value_ of the _expression_.
```

Comment:
You might want to say "and omit step 2" here?

Reply:
Or... "return the _fallback value_ as the _resolved value_ of the _expression_" (fast fail)